inbm-vision/Changelog.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## NEXT - ?
 
 ### Added
+ - RTC 517426 - Fixed for No post install check for network connectivity after post an OS update
```

Comment:
Can you instead say what was changed instead of that it was a fix.  Instead say, 'Network check added after reboot for SOTA'